src/Agent.Worker/Build/FileContainerServer.cs

```
@@ -184,6 +184,7 @@ public class FileContainerServer
             while (_fileUploadQueue.TryDequeue(out fileToUpload))
             {
                 token.ThrowIfCancellationRequested();
+                context.Debug(StringUtil.Loc("FileUploadCurrentFile", fileToUpload));
```

Should we use try-catch-rethrow instead of log every files?
otherwise the log might be to much noisy.

---

Agree, seems like only logging on error will be better.

---

👍